entrypoint.sh

```diff
@@ -14,10 +18,12 @@ fi
 # --- Determine docker host address ------------------------------------------
 
 function _resolve_host {
-  getent ahostsv4 "$1" | head -n1 | cut -d' ' -f1
+  ip=$(getent ahostsv4 "$1" | head -n1 | cut -d' ' -f1)
+  echo "$ip"
 }
 
-if [ "$DOCKER_HOST" ]
+# Check if the docker host env var is set
+if [ "${DOCKER_HOST}" ]
```

Comment:
because of `set -u` we need to have a default value e.g. `if [ "${DOCKER_HOST:-}" ]`
or even better  we set `DOCKER_HOST` in upfront at the top of the script e.g. `DOCKER_HOST="${DOCKER_HOST:-}"` to ensure it is always set, that way we don't default values for further access of `DOCKER_HOST`

Reply:
Indeed, I was indeed a bit too hasty with my changes.